apps/pie-storybook/.storybook/main.ts

```diff
@@ -14,7 +14,7 @@ const config: StorybookConfig = {
         options: {}
     },
     docs: {
-        autodocs: true
+        autodocs: false
```

Comment:
The auto-generated docs page wasn't being particularly useful, and the example components weren't displaying correctly, so I've removed this for now.